src/Hl7.Fhir.Support.Poco/Rest/EntryToTypedElementExtensions.cs

```diff
@@ -60,6 +46,8 @@ private static ITypedElement parseResource(string bodyText, string contentType,
                 throw new UnsupportedBodyTypeException(
                         "Endpoint said it returned '{0}', but the body is not recognized as either xml or json.".FormatWith(contentType), contentType, bodyText);
 
+
+            ITypedElement result;
```

Comment:
If you rewrite lines 68-71 as inline expression then you don't have to initialize result:

```
return  fhirType == ResourceFormat.Json 
                         ?  FhirJsonNode.Parse(bodyText).ToTypedElement(provider)
                         :  FhirXmlNode.Parse(bodyText).ToTypedElement(provider);
```